sourcegraph/sourcegraph
# [Search Jobs]: Add special zero state for the initial empty state

[Designs](https://www.figma.com/file/H4ezuXmEvVeoC6UaiNSbEa/Search-jobs-UI?type=design&node-id=1757-14620&mode=design&t=lcQ7R42aABKQGzA0-0)

This PR adds a special zero state when we have no filters and have no items in the search jobs list page. See screenshots below

| Type | Light | Dark | 
| --- | ------ | ------ |
| No items |  <img width="977" alt="Screenshot 2023-09-11 at 17 42 03" src="https://github.com/sourcegraph/sourcegraph/assets/18492575/32808499-6a7c-4c1f-bc16-62672a84340d"> | <img width="977" alt="Screenshot 2023-09-11 at 17 41 54" src="https://github.com/sourcegraph/sourcegraph/assets/18492575/616bb944-e014-4a39-b521-e8360396815b"> |
| No items because of filters | <img width="977" alt="Screenshot 2023-09-11 at 17 42 12" src="https://github.com/sourcegraph/sourcegraph/assets/18492575/02d7ca1c-af71-4425-bb3d-e67d41b52726"> | <img width="977" alt="Screenshot 2023-09-11 at 17 42 19" src="https://github.com/sourcegraph/sourcegraph/assets/18492575/1f6e9cea-81c9-4bc4-9b22-420faa201346"> |


## Test plan
- Run local sourcegraph client by running `sg start`
- Go to the `/search-jobs` (note that you have to have `searchJobs:true` in your experimental feature flags section in the settings)
- Check different zero states in both dark and light themes
